package.json

```diff
@@ -51,13 +55,22 @@
     "*.{ts,js,tsx,jsx,css,md}": "prettier --write"
   },
   "dependencies": {
+    "@babel/cli": "^7.12.1",
     "@babel/core": "^7.22.5",
     "@babel/generator": "^7.22.5",
+    "@babel/parser": "^7.26.9",
+    "@babel/plugin-transform-modules-commonjs": "^7.22.5",
+    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
     "@babel/template": "^7.22.5",
+    "@babel/traverse": "^7.26.9",
     "@babel/types": "^7.22.5",
     "@jest/types": "^29.6.3",
     "@swc/core": "^1.5.22",
     "@swc/jest": "^0.2.23",
+    "esbuild": "^0.18.0 || ^0.19.0 || ^0.20.0 || ^0.21.0 || ^0.22.0 || ^0.23.0 || ^0.24.0 || ^0.25.0",
```

Comment:
This looks pretty annoying to keep up to date.  Is the intention that each time esbuild releases a new version, this will be updated?  Why not use `>= 0.18.0`?